### ISSUE TITLE
Mobile: The "+" button doesn't work if pressed too slowly

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#11183]
### Operating system

Android

### Joplin version

3.0.8

### Current behaviour

See two of the comments [in this forum post](https://discourse.joplinapp.org/t/problem-right-at-the-beginning/41242/5?u=personalizedrefriger):
> I've also noticed that if I take longer than about half of a second to press the "+" button, nothing happens. Does tapping the "+" button quickly, with a smaller delay between touch start and end, help?

> The problem was indeed the duration of button press: It has to be done Ninja style, fast as a lightning. The problem is that I have an illness that slows down my speed.

### Expected behaviour

Long-pressing the "+" (and "edit") buttons should have an effect.
